### PR TITLE
Center settings nb

### PR DESCRIPTION
**Pull request by wski**

### CODE REVIEW

**Comment by wski** [comment]
@clarkjefcoat @blaino

File: app/js/components/management/mall/Intents.jsx [hunk 1]
```diff
@@ -97,13 +97,37 @@ var Intents = React.createClass({
             url: API_URL + '/api/intent',
             Schema: Intent,
             getDisplayName: function (selectedRecord) {
-                return `${selectedRecord.action}/${selectedRecord.type}`;
+                return `${selectedRecord.action}/${selectedRecord.mediaType}`;
+            },
+            structIcon: function(data){
+                data = data.map(function(intent) {
+                  var iconObj = {
```
Comment (blaino) [inline app/js/components/management/mall/Intents.jsx]: I understand wanting 2 spaces, but for existing files, we really should be matching the existing indentation.

File: app/js/components/management/mall/Stewards.jsx [hunk 2]
```diff
@@ -14,10 +14,37 @@ var Stewards = React.createClass({
...
             getDisplayName: function (selectedRecord) {
                 return selectedRecord.displayName;
             },
+            getUsername: (data) => {
+              return data.map((user)=> {
+                user.username = user.user.username;
+                return user;
+              });
+            },
+            getStweardOrgs: (data) => {
```
Comment (blaino) [inline app/js/components/management/mall/Stewards.jsx]: spelling

File: app/js/components/shared/Crud.jsx [hunk 3]
```diff
@@ -315,7 +327,20 @@ var Crud = React.createClass({
...
+                if(data.results[0].user) {
+                  this.props.getUsername(data.results);
+                }
+
+                if (data.results[0].icon) {
+                  data.results = this.props.structIcon(data.results);
+                }
+
+                if(data.results[0].stewardedOrganizations) {
+                  this.props.getStweardOrgs(data.results);
```
Comment (blaino) [inline app/js/components/shared/Crud.jsx]: spelling

**Comment by blaino** [comment]
Other than spelling and indentation, :+1:

**Comment by wski** [comment]
fixed

**Comment by clarkjefcoat** [comment]
:+1:

**Pull request merged**